chamge/amqp-arbiter-backend.c

```diff
@@ -107,18 +91,26 @@ _handle_edge_enroll (ChamgeAmqpArbiterBackend * self, JsonNode * root)
   JsonObject *json_object = NULL;
   const gchar *edge_id = NULL;
   ChamgeArbiterBackendClass *klass = CHAMGE_ARBITER_BACKEND_GET_CLASS (self);
+  gchar *body = NULL;;
```

Comment:
`g_autofree gchar *body = NULL;`

Then, we can be freed from worrying about `g_free()`.

Reply:
Yes.